packages/atlas/src/views/studio/VideoWorkspace/NftForm/NftForm.hooks.ts

```diff
@@ -6,7 +6,7 @@ import { Listing } from '@/views/studio/VideoWorkspace/NftForm/types'
 export const useNftForm = () => {
   const [activeInputs, setActiveInputs] = useState<string[]>([])
   const [termsAccepted, setTermsAccepted] = useState(false)
-  const [listingType, setListingType] = useState<Listing>(undefined)
+  const [listingType, setListingType] = useState<Listing>('Auction')
```

Comment:
This determines which listing type will be selected by default, right? Maybe would make sense to use "Not for sale" here to reduce friction if the user wants to just issue an NFT? It may be weird though since it's the last option. @KubaMikolajczyk any opinions?

Reply:
Actually now that we're moving issuance to a separate flow, maybe we don't need "Not for sale" option at all? Since now that flow will be about putting on sale, it assumes the NFT is not on sale atm

Reply:
So, should we remove "Not for sale" and stay with "Auction" as default selected option?

Reply:
@kdembler As a general rule of thumb, the fewer clicks and decisions in the process, the better. And so if you need a decision for now, I'd say it makes 100% sense to have the "Not for sale" option selected by default, as you suggested.

But then I think you're correct that we're most likely going to drop the "Not for sale" option with the upcoming design changes. And so if we're left with "Auction" and "Fixed price" as the only options to choose from, then my recommendation would be not to have anything selected by default.

Reply:
cc @KubaMikolajczyk

Reply:
Okay, to keep things simple let's leave "Auction" selected by default for now and don't worry too much about it @rafalpawlow, we can revisit in a future PR

Reply:
I don't know if we should change this to not for sale being a default - Im 100% for leaving Auction as a default mode and revisiting it in future when we get more data about frequency of user choice here